lib/viewmodels/posts_feed_view_model.dart

```diff
@@ -46,13 +46,9 @@ class PostsFeedViewModel extends AutoDisposeAsyncNotifier<List<PostDetails>> {
       position,
     );
 
-    // Wait for both futures to complete for optimisation
-    final List<dynamic> results = await Future.wait([
-      postsFirestoreFuture,
-      challengesFuture,
-    ]);
-    List<PostFirestore> postsFirestore = results[0];
-    final List<ChallengeFirestore> challenges = results[1];
+    List<PostFirestore> postsFirestore = await postsFirestoreFuture;
```

Comment:
This change decrease performance. The original author of this code wanted to have both future run at the same time asynchronously.

I think the solution for fixing the DCM hint here was to simply strip out the type and just have `final results = await Future.wait([` (if that works with type inference).

Reply:
Fixed with [ac35e5f](https://github.com/ProximaEPFL/proxima/pull/262/commits/ac35e5ffef1b4166248b9ad89b4c8509b076cac5)